src/common/components/null-component.tsx

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+
+export const NullComponent = NamedFC<{}>('NullComponent', () => {
```

Comment:
I don't think we need the `{}` on the props type template. Something like this should work just fine:
```typescript
export const NullComponent = NamedFC('NullComponent', () => null);
```